src/image/p5.Image.js

```diff
@@ -220,6 +220,32 @@ p5.Image = class {
     this.pixels = [];
   }
 
+  /**
+   * Set the pixel density of the image.
+   *
+   * @method setPixelDensity
+   * @param {Number} density - The pixel density to set.
+   * @example
+   * <div><code>
+   * let img = new p5.Image(100, 100);
+   * img.setPixelDensity(2);
+   * </code></div>
+   */
+  setPixelDensity(density) {
+    if (density <= 0) {
+      console.error('Pixel density must be greater than 0.');
```

Comment:
Instead of a `console.error` perhaps using the FES would be better here? Or alternatively default to 1 if the value is <= 0.

Reply:
I have implemented both of the suggested actions for improvement. Kindly review the changes and provide any feedback. Your input is highly valued. Thank you.